.gitlab-ci.yml

```diff
@@ -150,22 +203,3 @@ build-android:
   allow_failure:                   true
   <<:                              *collect_artifacts
 
-test-beta:
-  stage:                           optional
```

Comment:
why did we remove this?

Reply:
These two jobs were to test the code on `stable` Rust against `beta` and `nightly` Rust.
Developers answered they code only on `stable` Rust so it's not needed. What could be a bit more helpful is nightly build.